src/client/dfuse/ops/read.c

```diff
@@ -24,31 +24,80 @@
 #include "dfuse_common.h"
 #include "dfuse.h"
 
+#define READAHEAD_SIZE (1024*1024)
```

Comment:
Let's add a generic option to dfuse (e.g. -c or similar) where we enable all the caching/readahead/writeback stuff for applications that only do single-node access. We could also give it a try with etaluminis as well. Please add all this infrastructure and land this patch. There is definitely a use case for that.